packages/normalizr/package.json

```
@@ -29,6 +29,9 @@
   "module": "lib/index.js",
   "typings": "lib/index.d.ts",
   "sideEffects": false,
+  "engines": {
+    "node": ">=6.17.1"
```

```suggestion
    "node": ">=0.12"
```
apparently WeakMap is in node 0.12, so might as well have maximal back compat?